inc/class-wpseo-statistics.php

```diff
@@ -14,7 +14,7 @@ class WPSEO_Statistics {
 	 * @return int
 	 */
 	public function get_no_focus_post_count() {
```

Comment:
I feels like some overkill calling wrapper methods with the same endpoint: `get_post_count`.